assets/js/WorkflowList.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
```

Comment:
I love that you split this out into a separate file. I have some reservations about the name. We use "Workflow list" elsewhere (e.g. https://github.com/CJWorkbench/cjworkbench/blob/master/assets/css/style.scss#L1251) to refer to the /workflows page, that is, the list of the user's workflows. The list of modules within a Workflow we have called ModuleStack (e.g. https://github.com/CJWorkbench/cjworkbench/blob/master/assets/css/style.scss#L1313). So we need to either change the name elsewhere or here.